### PR TITLE
Updating Libraries

### PR DESCRIPTION
**Pull request by Rushi98**
Issue #92 
Updating Libraries
1. build tools -> `26.0.0`
2. checkstyle -> `7.8.1`
3. firebase -> `11.0.0`
4. support library -> `25.4.0`
5. realm -> `3.3.1`

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -19,6 +19,9 @@ allprojects {
     repositories {
         jcenter()
         mavenCentral()
+        maven {
+            url "https://maven.google.com"
```
Comment (kukreja-vikramaditya) [inline build.gradle]: Are you sure this is required?
Why wasn't this required earlier?

File: build.gradle [hunk 2]
```diff
@@ -8,7 +8,7 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:2.3.1'
         classpath 'com.google.gms:google-services:3.1.0'
-        classpath "io.realm:realm-gradle-plugin:3.3.0"
+        classpath "io.realm:realm-gradle-plugin:3.3.1"
```
Comment (kukreja-vikramaditya) [inline build.gradle]: remove unrelated changes.
Realm updates often so no need of staying ahead with each update.
Let us keep a target to update it when ver 3.5 is released

File: app/build.gradle [hunk 3]
```diff
@@ -5,23 +5,23 @@ apply plugin: 'realm-android'
 
 ext {
     androidTestVer = "0.5"
-    buildToolsVer = "25.0.2"
+    buildToolsVer = "26.0.0"
     butterknifeVer = "8.6.0"
-    checkstyleVer = '7.7'
+    checkstyleVer = '7.8.1'
```
Comment (kukreja-vikramaditya) [inline app/build.gradle]: Not relevant

File: build.gradle (hunk 1 above)
Comment (Rushi98) [inline build.gradle]: Yes. [setup instructions](https://developer.android.com/topic/libraries/support-library/setup.html)

Comment (kukreja-vikramaditya) [inline build.gradle]: Ok I am fine with it . Please make the remaining changes and we are good to go
